lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1551,21 +1551,80 @@ func (w *WalletKit) fundPsbtInternalWallet(account string,
 				return err
 			}
 
+			// filterFn makes sure utxos which are unconfirmed and
+			// still used by the sweeper are not used.
+			filterFn := func(u *lnwallet.Utxo) bool {
+				if u.Confirmations == 0 &&
```

Comment:
nit: can check `if u.Confirmations != 0` to exit early

Reply:
changed